apps/frontend/src/views/Login.vue

```diff
@@ -28,12 +28,10 @@
                     id="password_field"
                     v-model="password"
                     :error-messages="passwordErrors"
+                    type="password"
                     name="password"
                     label="Password"
                     prepend-icon="mdi-lock"
-                    :type="showPassword ? 'text' : 'password'"
-                    :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
-                    @click:append="showPassword = !showPassword"
```

Comment:
Since `show_secret` is no longer used, the data field can be removed from this Component.

Reply:
I've removed showPassword from Login.vue already, are you referring to something else?

Reply:
It unfortunately won't let me comment on the correct line, on `AuthStep.vue` and `AuthStepBasic.vue` there is still a `show_secret: boolean = false;` that is unused.

https://github.com/mitre/heimdall2/blob/35bbbfc7980c10f60cd6173e759a6d694d3f4e8e/apps/frontend/src/components/global/upload_tabs/splunk/AuthStep.vue#L68
https://github.com/mitre/heimdall2/blob/35bbbfc7980c10f60cd6173e759a6d694d3f4e8e/apps/frontend/src/components/global/upload_tabs/aws/AuthStepBasic.vue#L73